### DISCUSSION TITLE
Is there a way to add custom content (bar, modal, etc...) and call nuxt-cookie-control actions?

### DISCUSSION THREAD

**Discussion by GuillaumeDgr** [dargmuesli/nuxt-cookie-control#92] [Q&A] [answered]
### Describe the feature

Set my own design modal and bar and call show action modal action, decline cookies, accept, etc... from my own comonents?

### Additional information

- [ ] Would you be willing to help implement this feature?

### Final checks

- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by dargmuesli** [comment]
Yes, please check the README: https://github.com/dargmuesli/nuxt-cookie-control#usage

```ts
const {
  cookiesEnabled,
  cookiesEnabledIds,
  isConsentGiven
} = useCookieControl()
```

**Reply by kogratte** [reply]
Stop me if I'm wrong, but that does not allow us to implement anything custom. There is no way to replace buttons, use another framework modal and so on and so on.

**Reply by dargmuesli** [reply, marked as answer]
It does, but only as full replacement. It's not possible to replace single buttons, but you could create a custom modal, taking inspiration from [CookieControl.vue](https://github.com/dargmuesli/nuxt-cookie-control/blob/09f8d0b48a12682d7bbfd0a163abafeae264cc51/src/runtime/components/CookieControl.vue), and write the result into the variables mentioned above.
